README.md

```diff
@@ -97,6 +97,14 @@ rhc_repositories:
 
 A release to set for the system. Use `{"state":"absent"}` to actually unset the
 release set for the system.
+```
+rhc_insights:
+  state: present
+```
+
+Whether the system is connected to Insights; valid values are `present`
+(to ensure registration/connection), and `absent`.
```

Comment:
What if the user does not specify `rhc_insights` at all?  What should the role do?  All of the `rhc_insights: state: absent` added below suggests that the role will **always** attempt to change the insights registration status.  Is this really what we want?  I mean, if the default is `present`, then the role will always attempt to register with insights, *even if the user does not want that*, even if `rhc_insights` is not specified.  Conversely, if the default is `absent`, then the role will always attempt to unregister with insights if `rhc_insights` is not specified.

I think the role should do like @ptoscano did elsewhere:
* if `rhc_insights` is not set i.e. `rhc_insights == {}`, do nothing - do not attempt to register or unregister
* if `rhc_insights` has `state: present` - register
* if `rhc_insights` has `state: absent` - unregister

Reply:
I think this is a better option. I was strictly using the definition we made in the document, I'll update that too.

Reply:
yes, sounds good

Reply:
Just a note to this. The tests that didn't use insights-client won't try to unregister insights as it is not registered.

>rhc_state=present (specified or default): rhc_insights.state can be anything (defaults to present, so it will either 
> register to Insights or not, still with a registered-to-rhsm system
> rhc_state=absent (specified):
> rhc_insights.state ought to default to absent
> rhc_insights.state=present (specified) ought to be rejected
> rhc_insights.state=absent (specified) is OK

I think this is correct.